fieldservice/models/fsm_order.py

```diff
@@ -40,14 +40,17 @@ def _default_team_id(self):
         else:
             raise ValidationError(_("You must create an FSM team first."))
 
-    @api.depends("date_start", "date_end")
+    api.depends("date_start", "date_end")
```

Comment:
```suggestion
    @api.depends("date_start", "date_end")
```
(and remove blank line)